src/itp.rst

```diff
@@ -0,0 +1,218 @@
+
+.. container:: bib-item
+
+  .. bibliography:: itp.bib
+    :filter: key == 'lerner15'
+
+  Polymorphic Blocks is purely graphical, and allows users to 
+  prove simple formulas in propositional logic. The idea is to use shapes to represent boolean connectives and
+  variables, blocks to represent inference rules and theorems, and composition of blocks to represent formal deductions.
+
+  In their evaluation, they recruited about 30 high school students who have no experience with logic, and randomly divided into two
+  groups; in the symbol group, students are required to do formal proofs about propositional logic, while in the game group, they are
+  required to do the exact same problem sets in the Polymorphic Blocks game. The average completion time of the game group is about half
+  of the symbolic group. After that, each group is given the same additional problem set, and participants are asked to work on that for
+  at least 5 minutes. They observed that in the game group, everyone finished the problems, while in the symbol group, most of them gave
+  up in the middle.
+
+  This paper concludes that the puzzle game like tool helps users to do proof much faster than a pen-and-paper interface.
+  It also results in more accuracy and engagement. The tool has a potential of teaching
+  logic, however the tool only teaches the user strategies of applying inference rules without going into details, and it is not clear
+  whether it helps the user understand the symbolic meaning of the formulas. The paper also states that this tool has a potential of
+  laying the foundation for crowdsourcing proofs via gamification.
+
+Proof General
+=============
+.. container:: bib-item
+
+  .. bibliography:: itp.bib
+    :filter: key == 'aspinall00'
+
+  Proof General is a generic interface for various theorem provers based on Emacs. It features in simplified communication with the 
+  theorem prover by providing a GUI. Scripts displayed by the GUI are syncronized with the prover. User can edit the proof
+  stripts, see the current goal, and receive feed back from the theorem prover.
+
+.. container:: bib-item
+
+  .. bibliography:: itp.bib
+    :filter: key == 'aspinall07'
+
+  Proof General Kit implements a protocol for communicating between
+  theorem provers, display and other auxiliary tools, and it integrates features such as browsing and searching loaded theorems.
+
+
+Now there are `more features, <http://proofgeneral.inf.ed.ac.uk/htmlshow.php?title=Proof+General+user+manual&file=releases%2FProofGeneral%2Fdoc%2FProofGeneral%2FProofGeneral_2.html>`_
+supported by proof general, such as proof by pointing (appropraite tactics or lemmas are automatically applied
+when clicking on a subterm of a goal) and proof-tree visualization.
+
+
+Geometry Interactive Theorem Prover (GTP)
+=========================================
+
+
+GTPs has potential for education. 
```

Comment:
What is GTP? You need to introduce the topic before saying it has potential for education.